make/buf/all.mk

```diff
@@ -116,7 +116,7 @@ bufgeneratesteps:: \
 
 .PHONY: bufrelease
 bufrelease: $(MINISIGN)
-	DOCKER_IMAGE=golang:1.19.5-bullseye bash make/buf/scripts/release.bash
+	DOCKER_IMAGE=golang:1.20-bullseye bash make/buf/scripts/release.bash
```

Comment:
There is a `1.20.0-bullseye` as well, and we traditionally have locked to micros, but with our workflows for Docker, I don't think this actually makes as much sense. We may want to just lock to minors for Docker going forward, so we always push the latest.